docroot/modules/custom/sitenow_migrate/src/Plugin/migrate/source/BaseNodeSource.php

```diff
@@ -106,7 +109,7 @@ protected function extractSummaryFromText(string $text) {
     // that might cause weird spacing for our summaries.
     $new_summary = ltrim(strip_tags($new_summary));
 
-    $new_summary = substr($new_summary, 0, 200);
+    $new_summary = substr($new_summary, 0, 400);
```

Comment:
Upping the length to 400 from the small 200 takes care of the test case, but guarantee a similar one won't pop up again, given a wordy-enough intro.

Reply:
Oh I see. So if a sentence or breaking point or whatever is over the limit, it returns nothing?

Reply:
Oh yea they want an ellipsis if its over the limit in #3949... Maybe we could try out smart_trim in the update hook for that issue to see how smart it really is and then come back to this. We can refactor these tests a bit to better cover what we're trying to ensure works.